src/Util.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace React\Http;
+
+/**
+ * @internal
+ */
+final class Util
```

Comment:
Not a big fan of a "utility" class, but given this is `@internal` only, I won't block this and would rather clean this up in a follow-up PR :+1: 

Does it make sense to move this to `Io` namespace so this is less prominent?

Reply:
Neither am I, another option would be to create a functions file. Moved it under `Io`